app/assets/stylesheets/modules/clipboard.css

```diff
@@ -0,0 +1,33 @@
+.clipboard--otp {
```

Comment:
Tried my best to follow the [BEM naming conventions](https://getbem.com/naming/) that is used in the codebase.